src/routes/index.svelte

```diff
@@ -1,4 +1,6 @@
 <script>
+	/* Importing the image rather than just specifying the URL as `src` will give it a URL containing a hash, 
```

Comment:
I personally think that the explanation on the README is probably sufficient. In my mind, it's probably better not to add documentation in comments like this because it's something that most users will end up having to remove after cloning the template, so it's better not to have it scattered throughout

Reply:
I think it's pretty likely people will not read the README; it would be good to have the explanation nearby the code it refers to because that is where they will start wondering what the code does. 

You're right this is a template; on the other hand they will most definitely remove the successkid anyway, so they might as well remove the comment too. But sure, I can remove it.